### PR TITLE
Detail view for facilitator application

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx [hunk 1]
```diff
@@ -4,63 +4,176 @@ import LabeledFormComponent from "../../form_components/LabeledFormComponent";
...
+                  Please note that we prioritize applicants in regions where we currently have a
+                  Regional Partner supporting CS Fundamentals, and there is a need for additional
+                  facilitators. Code.org will review your application and contact you if there is
+                  a need for facilitators. We are not able to guarantee a space for you in a
+                  different location.
+                </p>
+              </div>
+            }
+            {
+              this.props.data.regionalPartnerId && !PartnersWithoutCsf.includes(this.props.data.regionalPartnerId) &&
```
Comment (mehalshah) [inline apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx:157]: The way that we've implemented PartnersWithoutCsf is dodgy - we really shouldn't have hardcoded IDs anywhere. Instead, this would have been better implemented as an attribute on the regional partner.

We'll have to take this as a tech debt work item

File: apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+      </FormGroup>
+    );
+  }
+
+  /**
+   * @override
+   */
+  static getDynamicallyRequiredFields(data) {
+    const requiredFields = [];
+    const program = data.program || 'CS Program';
```
Comment (mehalshah) [inline apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx:251]: Why do we default this to 'CS Program'?

File: apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx [hunk 3]
```diff
@@ -73,7 +73,7 @@ export default class AdminCohortView extends React.Component {
       date_accepted: 'Date Accepted',
       applicant_name: 'Name',
       district_name: 'School District',
-      school_name: 'School Name',
+      school_name: 'School Name 1',
```
Comment (mehalshah) [inline apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx]: Why this change?

File: apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx [hunk 4]
```diff
@@ -73,7 +73,7 @@ export default class DetailViewApplicationSpecificQuestions extends React.Compon
     if (!this.state.principalApproval) {
       return (
         <div>
-          <h4>Select option</h4>
+          <h4>Select otion</h4>
```
Comment (mehalshah) [inline apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx]: Should be option

File: dashboard/app/controllers/api/v1/pd/applications_controller.rb [hunk 5]
```diff
@@ -201,6 +205,8 @@ def update
       end
 
       unless @application.update(application_data)
+        p "Hit 'unless' condition"
```
Comment (mehalshah) [inline dashboard/app/controllers/api/v1/pd/applications_controller.rb]: Remove this

**Review by mehalshah — COMMENTED** [review]
Some comments to address

File: apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx (hunk 3 above)
Comment (jmkulwik) [reply apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx]: Typo

File: apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx [hunk 6]
```diff
@@ -4,63 +4,176 @@ import LabeledFormComponent from "../../form_components/LabeledFormComponent";
...
+      this.loadPartnerRequest = null;
 
-            {this.props.data.csfAvailability === CSF_AVAILABILITY_ONLY_WEEKEND &&
-              this.inputFor("csfPartialAttendanceReason", this.indented())
+      this.handleChange({
+        regionalPartnerId: data.id,
+        regionalPartnerName: data.name
+      });
+
+      // Update state with all the partner workshop data to display
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx]: this comment doesn't make sense here anymore

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx [hunk 7]
```diff
@@ -817,7 +1017,7 @@ export class DetailViewContents extends React.Component {
       return (
         <div>
           <h3>Principal Approval</h3>
-          <h4>Select option</h4>
+          <h4>Select One</h4>
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx]: Should be "Select option"

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.story.jsx [hunk 8]
```diff
@@ -20,7 +20,7 @@ export default storybook => {
               regional_partner_name: 'partner',
               notes: 'notes',
               status: 'unreviewed',
-              school_name: 'School Name',
+              school_name: 'School Name 1',
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view_contents.story.jsx:23]: Should be "School Name"

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx [hunk 9]
```diff
@@ -643,6 +788,51 @@ export class DetailViewContents extends React.Component {
...
+  renderInterview = () => {
+    let interviewFields = [];
+    [
+      {label: 'question1', id: 'question_1', value: this.state.question_1},
+      {label: 'question2', id: 'question_2', value: this.state.question_2},
+      {label: 'question3', id: 'question_3', value: this.state.question_3},
+      {label: 'question4', id: 'question_4', value: this.state.question_4},
+      {label: 'question5', id: 'question_5', value: this.state.question_5},
+      {label: 'question6', id: 'question_6', value: this.state.question_6},
+      {label: 'question7', id: 'question_7', value: this.state.question_7}
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx:800]: Individually enumerating all of these questions is also adding tech debt and should be fixed. These should be passed as an arrary/list instead of 7 separate items.

File: dashboard/app/controllers/api/v1/pd/applications_controller.rb [hunk 10]
```diff
@@ -201,6 +205,8 @@ def update
...
+        p @application.errors.full_messages
```
Comment (jmkulwik) [inline dashboard/app/controllers/api/v1/pd/applications_controller.rb]: This should also probably be removed

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx (hunk 9 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx:800]: It's not that bad in terms of the debt added - we're shaving a few lines of code but this is a lower priority than hardcoded regional partner IDs

File: apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx [hunk 11]
```diff
@@ -4,63 +4,176 @@ import LabeledFormComponent from "../../form_components/LabeledFormComponent";
...
+              <div>
+                <p>
+                  <strong>Your Regional Partner is not accepting applications for CS Fundamentals facilitators at this time.</strong>
+                </p>
+                <p>
+                  Please note that we prioritize applicants in regions where we currently have a
+                  Regional Partner supporting CS Fundamentals, and there is a need for additional
+                  facilitators. Code.org will review your application and contact you if there is
+                  a need for facilitators. We are not able to guarantee a space for you in a
+                  different location.
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx:152]: It seems like these strings should be in a constants/strings file or I18N file somewhere. Is there a plan to migrate?

File: apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx [hunk 12]
```diff
@@ -0,0 +1,295 @@
...
+      });
+    }).error(() => {
+      this.setState({
+        loadingSummerWorkshops: false,
+        loadError: true
+      });
+    });
+  }
+
+  renderCsdCspWhichFitWeekend() {
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx:113]: Bit confusing from the title what is happening here. Suggestions for rename:
renderCsdCspFitWeekendSelector
renderCsdCspFitWeekendOptions
renderFitWeekendOptions

File: apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx [hunk 13]
```diff
@@ -0,0 +1,295 @@
...
+
+      return this.dynamicCheckBoxesWithAdditionalTextFieldsFor(
+        "csdCspWhichFitWeekend",
+        options,
+        textFieldMap
+      );
+    }
+  }
+
+  renderCsdCspWhichSummerWorkshop() {
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx:133]: Also suggest renaming

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 14]
```diff
@@ -0,0 +1,229 @@
...
+        </p>
+        <p>
+          5: Clearly articulates the importance of CS accessibility for all teachers
+          regardless of background or certification.
+        </p>
+      </div>
+    )
+  },
+  skillsAreasToImprove: {
+    title: 'Growth minded',
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: this is capitalized everywhere else (`'Growth Minded'`)

File: apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx [hunk 15]
```diff
@@ -0,0 +1,295 @@
...
+  renderCsdCspWhichFitWeekend() {
+    if (this.props.data.fitWorkshops && this.props.data.fitWorkshops.length > 0) {
+      const options = this.props.data.fitWorkshops.map(workshop =>
+        `${workshop.dates} in ${workshop.location}`
+      );
+      options.push(TextFields.notSurePleaseExplain, TextFields.unableToAttendPleaseExplain);
+
+      const textFieldMap = {
+        [TextFields.notSurePleaseExplain] : "not_sure",
+        [TextFields.unableToAttendPleaseExplain] : "unable_to_attend"
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx:122]: "not_sure" and "unable_to_attend" are used in multiple places. Perhaps move to variable

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 16]
```diff
@@ -0,0 +1,229 @@
...
+  },
+  inquiryBasedLearning: {
+    title: 'Teaching Experience',
+    rubric: (
+      <div>
+        <p>
+          0: Does not provide a clear definition or example of inquiry-based learning
+        </p>
+        <p>
+          3: Defines inquiry-based learning but has not led an inquiry-based activity.
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: inconsistent to end only this one with a period and not the others

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 17]
```diff
@@ -0,0 +1,229 @@
...
+        <p>
+          0: Describes neither what they want to learn nor the impact they want to make,
+          OR states that they want to learn because their intention is to become a trainer
+          primarily for their district
+        </p>
+        <p>
+          3: Describes either what they want to learn OR the impact they want to make
+        </p>
+        <p>
+          5: Describes what they want to learn AND the impact they want to make.
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: inconsistent to end only this one with a period and not the others

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 18]
```diff
@@ -0,0 +1,229 @@
...
+  },
+  whyShouldAllHaveAccess: {
+    title: 'Equity',
+    rubric: (
+      <div>
+        <p>
+          0: Does not articulate why CS should be accessible to all teachers
+        </p>
+        <p>
+          3: Articulates the importance of CS, but is not clear about accessibility for
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx:79]: inconsistent to end this and the next one with periods but not the first one.

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 19]
```diff
@@ -0,0 +1,229 @@
...
+        </p>
+      </div>
+    )
+  },
+  question_2: {
+    title: 'Equity',
+    rubric: (
+      <div>
+        <p>
+          0: No, or unclear, strategies for supporting teachers.
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: should this be "engaging and supporting"?

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 20]
```diff
@@ -0,0 +1,229 @@
...
+        </p>
+      </div>
+    )
+  },
+  question_3: {
+    title: 'Growth Minded',
+    rubric: (
+      <div>
+        <p>
+          0: Not able to clearly describe how they would respond, take next steps or how
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx:177]: should end with a period for consistency

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 21]
```diff
@@ -0,0 +1,229 @@
...
+        </p>
+      </div>
+    )
+  },
+  question_4: {
+    title: 'Growth Minded',
+    rubric: (
+      <div>
+        <p>
+          0: Not able to clearly describe how they would respond, take next steps or offer
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx:196]: should end with a period for consistency

File: apps/src/code-studio/pd/application/facilitator1920/Section6Submission.jsx [hunk 22]
```diff
@@ -1,23 +1,34 @@
...
   static associatedFields = [
-    ...Object.keys(PageLabels.section7Submission)
+    ...Object.keys(PageLabels.section6Submission)
   ];
 
   render() {
     return (
       <FormGroup>
-        <h3>Section 7: {SectionHeaders.section7Submission}</h3>
+        <h3>Section 6: {SectionHeaders.section6Submission}</h3>
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application/facilitator1920/Section6Submission.jsx:20]: Having these numbers hard-coded in multiple separate files makes me nervous. Can we add to our tech debt to find a way to organize this so we aren't dependent on manually updating the numbers?

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 23]
```diff
@@ -0,0 +1,229 @@
...
+export const FacilitatorScoringFields = {
+  currentlyInvolvedInCsEducation: {
+    title: 'Champion for CS',
+    rubric: (
+      <div>
+        <p>
+          0: Does not currently support CS education
+        </p>
+        <p>
+          3: Supports CS education but not as an educator- Tech employee who volunteers in
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: educator-
should be
educator -

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 24]
```diff
@@ -0,0 +1,229 @@
...
+  },
+  gradesTaught: {
+    title: 'Teaching Experience',
+    rubric: (
+      <div>
+        <p>
+          0: No experience at the K-12 level
+        </p>
+        <p>
+          3: K-12 experience, but not at the same level as the program they’re applying to
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: remove smart quote. i.e. they're

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 25]
```diff
@@ -0,0 +1,229 @@
...
+          lead
+        </p>
+        <p>
+          5: Experience at the same level as the program they’re applying to lead
```
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: smart quote

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx (hunk 17 above)
Comment (jmkulwik) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: minor comment: there is inconsistent use of periods at the end of sentences for grading. Do we want there to always or never be a period?

File: dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb [hunk 26]
```diff
@@ -12,12 +12,12 @@ def new_form
...
 
     def on_successful_create
-      @application.assign_default_workshop!
-      @application.assign_default_fit_workshop!
-      fit_workshop = @application.find_default_fit_workshop
-      @application.fit_workshop_id = fit_workshop.id if fit_workshop
+      # @application.assign_default_workshop!
+      # @application.assign_default_fit_workshop!
+      # fit_workshop = @application.find_default_fit_workshop
+      # @application.fit_workshop_id = fit_workshop.id if fit_workshop
```
Comment (jmkulwik) [inline dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb]: Should these lines be removed rather than commented?

File: dashboard/app/models/pd/application/facilitator1819_application.rb [hunk 27]
```diff
@@ -50,12 +50,32 @@ def year
...
+      super.merge(
+        {
+          csd_csp_teachercon_availability: [
+            'TeacherCon 1: June 17 - 22, 2018',
+            'TeacherCon 2: July 22 - 27, 2018',
+            TEXT_FIELDS[:not_available_for_teachercon]
+          ],
+          csd_csp_fit_availability: [
+            'June 23 - 24, 2018 (immediately following TeacherCon 1)',
+            'July 28 - 29, 2018 (immediately following TeacherCon 2)',
```
Comment (jmkulwik) [inline dashboard/app/models/pd/application/facilitator1819_application.rb:72]: Seems like these strings should be in the constants file

**Review by jmkulwik — APPROVED** [review]
Overall looks good! There are a couple items I would recommend adding to tech debt backlog (see comments). I didn't take a close look at the test cases (let me know if you'd like me to take a closer look there). One thing I am wondering is how long we plan to maintain the old applications i.e. 1819. Will we eventually remove them?

File: apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx (hunk 12 above)
Comment (clareconstantine) [reply apps/src/code-studio/pd/application/facilitator1920/Section3ExperienceAndCommitments.jsx:113]: It renders the question whose label key is csdCspWhichFitWeekend, which is the naming convention I'm using throughout the application. It's not a different kind of question in the ui (just checkboxes), but we are dynamically creating the options based on the workshops that are in the workshop dashboard at the time the application is being filled out. I can add a comment to explain the dynamic options, if that would help?

File: dashboard/test/controllers/pd/application/facilitator_application_controller_test.rb [hunk 28]
```diff
@@ -63,7 +63,7 @@ class FacilitatorApplicationControllerTest < ::ActionController::TestCase
       sign_in application.user
       get :new
       assert_response :success
-      assert_select 'p', /Your application is not yet matched to a Code\.org Regional Partner\./
+      assert_select 'p', /^As a reminder, your application is not yet matched to a Code.org Regional Partner/
```
Comment (clareconstantine) [inline dashboard/test/controllers/pd/application/facilitator_application_controller_test.rb]: remove carat

File: dashboard/test/factories/pd_factories.rb [hunk 29]
```diff
@@ -941,30 +941,49 @@
...
+      csf_good_standing_requirement Pd::Application::Facilitator1920Application::YES_COMMIT
+      csf_summit_requirement Pd::Application::Facilitator1920Application::YES_COMMIT
+      csf_workshop_requirement Pd::Application::Facilitator1920Application::YES_COMMIT
+      csf_community_requirement Pd::Application::Facilitator1920Application::YES_COMMIT
     end
 
     trait :with_csd_csp_specific_fields do
-      csd_csp_fit_availability Pd::Application::Facilitator1920Application.options[:csd_csp_fit_availability].first
-      csd_csp_teachercon_availability Pd::Application::Facilitator1920Application.options[:csd_csp_teachercon_availability].first
+      csd_csp_no_partner_summer_workshop Pd::Application::Facilitator1920Application.options[:csd_csp_no_partner_summer_workshop].first
```
Comment (clareconstantine) [inline dashboard/test/factories/pd_factories.rb:983]: add `csd_csp_good_standing_requirement` and double check against final version of required fields

File: dashboard/test/models/pd/application/facilitator1920_application_test.rb [hunk 30]
```diff
@@ -105,10 +108,18 @@ class Facilitator1920ApplicationTest < ActiveSupport::TestCase
...
-      refute answers.key? :csd_csp_fit_availability
-      refute answers.key? :csd_csp_teachercon_availability
+      [
+        :csf_good_standing_requirement,
+        :csf_summit_requirement,
+        :csf_workshop_requirement,
+        :csf_community_requirement
+      ].each {|x| assert answers.key? x}
+      [
+        :csd_csp_lead_summer_workshop_requirement,
```
Comment (clareconstantine) [inline dashboard/test/models/pd/application/facilitator1920_application_test.rb:119]: add `csd_csp_good_standing_requirement` and double check against final version of required fields

File: dashboard/test/models/pd/application/facilitator1920_application_test.rb [hunk 31]
```diff
@@ -119,10 +130,19 @@ class Facilitator1920ApplicationTest < ActiveSupport::TestCase
...
-        assert answers.key? :csd_csp_teachercon_availability
+
+        [
+          :csf_good_standing_requirement,
+          :csf_summit_requirement,
+          :csf_workshop_requirement,
+          :csf_community_requirement
+        ].each {|x| refute answers.key? x}
+        [
+          :csd_csp_lead_summer_workshop_requirement,
```
Comment (clareconstantine) [inline dashboard/test/models/pd/application/facilitator1920_application_test.rb:143]: add `csd_csp_good_standing_requirement` and double check against final version of required fields

File: lib/cdo/shared_constants/pd/facilitator1920_application_constants.rb [hunk 32]
```diff
@@ -1,11 +1,200 @@
...
+      ],
+      program_commitment_score: [
+        :why_interested
+      ]
+    }
+
+    INTERVIEW_QUESTIONS = {
+      question_1: clean_multiline(
+        'Question 1: Code.org’s programs are designed for teachers of all backgrounds, especially teachers with
+        little-to-no experience with computer science. What are two strategies you would use to engage and support
```
Comment (clareconstantine) [inline lib/cdo/shared_constants/pd/facilitator1920_application_constants.rb]: "Little to no" isn't usually hyphenated

File: lib/cdo/shared_constants/pd/facilitator_common_application_constants.rb [hunk 33]
```diff
@@ -7,168 +7,148 @@ def self.clean_multiline(string)
...
+        csf_workshop_requirement: 'Can you commit to facilitating a minimum of 4-6 one-day workshops starting summer 2019 and continuing throughout the 2019-2020 school year?',
+        csf_community_requirement: 'Can you commit to attending monthly webinars, or watching recordings, and staying up to date through bi-weekly newsletters and online facilitator communities?',
+        csd_csp_fit_weekend_requirement: 'Can you commit to attending one 3-day 2019 Facilitator-in-Training Workshop?',
+        csd_csp_which_fit_weekend: 'Please indicate which FiT Workshop you are able to attend.',
+        csd_csp_workshop_requirement: 'Can you commit to leading 2019-20 academic year workshops hosted by your assigned Regional Partner (generally four days across the academic year)?',
+        csd_training_requirement: 'Can you commit to attending __one__ 2- or 3-day training in the spring of 2020 to prepare to co-lead five-day summer workshops?',
+        csp_training_requirement: 'Can you commit to attending __two__ 2- or 3-day trainings in the spring of 2020 to prepare to co-lead five-day summer workshops?',
```
Comment (clareconstantine) [inline lib/cdo/shared_constants/pd/facilitator_common_application_constants.rb]: In the above two questions, should be "5-day" for consistency with the rest of the application

File: dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb (hunk 26 above)
Comment (clareconstantine) [reply dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb]: we can remove them - I have a checklist item to update this to the new way they want this to work this year.

File: lib/cdo/shared_constants/pd/facilitator_common_application_constants.rb [hunk 34]
```diff
@@ -7,168 +7,148 @@ def self.clean_multiline(string)
...
+        csf_workshop_requirement: 'Can you commit to facilitating a minimum of 4-6 one-day workshops starting summer 2019 and continuing throughout the 2019-2020 school year?',
+        csf_community_requirement: 'Can you commit to attending monthly webinars, or watching recordings, and staying up to date through bi-weekly newsletters and online facilitator communities?',
+        csd_csp_fit_weekend_requirement: 'Can you commit to attending one 3-day 2019 Facilitator-in-Training Workshop?',
+        csd_csp_which_fit_weekend: 'Please indicate which FiT Workshop you are able to attend.',
+        csd_csp_workshop_requirement: 'Can you commit to leading 2019-20 academic year workshops hosted by your assigned Regional Partner (generally four days across the academic year)?',
+        csd_training_requirement: 'Can you commit to attending one 5-day training in the spring of 2020 to prepare to co-lead five-day summer workshops?',
+        csp_training_requirement: 'Can you commit to attending __two__ 5-day trainings in the spring of 2020 to prepare to co-lead five-day summer workshops?',
```
Comment (clareconstantine) [inline lib/cdo/shared_constants/pd/facilitator_common_application_constants.rb]: I should have been more specific in my previous comment. These should go back to saying “__ one __ 2- or 3-day training” and “__ two __ 2- or 3-day trainings” respectively. I just meant that instead of “five-day summer workshops” (later in the question) we should write it “5-day summer workshops” because that is how we refer to summer workshops everywhere else.

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx [hunk 35]
```diff
@@ -0,0 +1,229 @@
...
+        </p>
+      </div>
+    )
+  },
+  question_2: {
+    title: 'Equity',
+    rubric: (
+      <div>
+        <p>
+          0: No, or unclear, strategies for engaging supporting teachers.
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: Missing “and” - should say “engaging and supporting”

File: apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx (hunk 11 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx:152]: Not right now. Applications and pretty much all PD is english only. We may have to revisit this going forward but for now, we're keeping it simple.

File: apps/src/code-studio/pd/application/facilitator1920/Section6Submission.jsx (hunk 22 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application/facilitator1920/Section6Submission.jsx:20]: We could probably update this line to infer the number from the section header name. However, in terms of tech debt priority, this is relatively low. These files don't get reordered frequently.

File: apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx (hunk 17 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx]: I'll remove all periods

File: dashboard/app/models/pd/application/facilitator1819_application.rb (hunk 27 above)
Comment (mehalshah) [reply dashboard/app/models/pd/application/facilitator1819_application.rb:72]: These aren't shared constants - keeping them here is fine

**Review by clareconstantine — APPROVED** [review]
